meh/handler.py

```diff
@@ -20,7 +20,14 @@
 import bdb
 import os
 import sys
-import report
+
+LIBREPORT_AVAILABLE = False
+try:
+    import report
+    LIBREPORT_AVAILABLE = True
+except ImportError:
+    print("python-meh: libreport is not available in this environment")
```

Comment:
I guess you want to print this into stderr.

Reply:
Yeah, lets do that.